modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/utils.ts

```diff
@@ -369,29 +369,20 @@ export function getUnsupportedObjectRelationshipErrorMessage(
 	}
 }
 
-export function updatePreviousURLParam(paramType: string, paramValue: string) {
-	const previousPath = document.referrer;
-
-	const newPreviousURL = new URL(previousPath);
-
-	const objectFolderNameParam = newPreviousURL.searchParams.get(paramType);
-
-	if (objectFolderNameParam) {
-		newPreviousURL.searchParams.set(paramType, paramValue);
-
-		window.history.pushState(null, '', newPreviousURL.toString());
-
-		window.location.href = newPreviousURL.toString();
-	}
+interface updatePreviousURLParam {
```

Comment:
Hey @rebsilva and @igor-franca, just a small change, there is a [naming convention for interfaces in TypeScript](https://aykhanhuseyn.medium.com/typescript-naming-conventions-crafting-maintainable-code-7d872234fe17) and it says that the interface name must adhere to the PascalCase convention, which means that the first letter of each word in the name is capitalized. 

I saw that we have two other interfaces that are outside this convention in this file (`getObjectDefinitionNodePosition` and `handleUnplacedObjectDefinitionNode`), could you fix that too?